### DISCUSSION TITLE
Could we have buttons to clear Ray?

### DISCUSSION THREAD

**Discussion by ckilb** [spatie/ray#732] [Ideas]
It's nice that we can set up a keyboard shortcut to clear Ray from everywhere. 
But once in a while this doesn't seem to work & in general I'm more a mouse guy. 

Would it be possible to add a small button to clear the debug logs?

**Comment by freekmurze** [comment]
Hi @ckilb

You can clear the screen by clicking that "+" button in the menu bar.

**Reply by ckilb** [reply]
Thanks! I was expecting some trash icon or similar, but now I understand the purpose of different screens. Thanks!
